modules/openapi-generator/src/main/resources/typescript-angular/package.mustache

```diff
@@ -11,7 +11,7 @@
     "openapi-client",
     "openapi-generator"
   ],
-  "license": "Unlicense",
+  "license": "{{licenseName}}",
```

Comment:
lets only add it when a value is set, same in the other generators

```suggestion
  {{#licenseName}}
  "license": "{{licenseName}}",
  {{/licenseName}}
```

Reply:
The reason why I kept it like this for some and not for others is because some will always have a default value set. IDK if it is possible for this value to be unset for generators that have the default value set in the field directly.

Reply:
I guess one can explicitly set to an empty value, then it should not pass the `{{#licenseName}}` condition

Reply:
I wasn't able to do this with the cli if I set the licenseName to empty value it will still be present. I tested this on this branch with `typescript-fetch` for which I have this in place already:
```
❯ java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g typescript-fetch -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -t modules/openapi-generator/src/main/resources/typescript-fetch -o samples/client/petstore/typescript-fetch/builds/default --additional-properties npmName=npmName,licenseName=
❯ grep "license" samples/client/petstore/typescript-fetch/builds/default/package.json
  "license": "",
❯ java -jar ... generate ... licenseName="" > /dev/null && grep "license" ...package.json
  "license": "",
❯ java -jar ... generate ... licenseName="null" > /dev/null && grep "license" ...package.json
  "license": "null",
```

Maybe there is some other way to set it to null, but I was not able to do it.

I could still implement it, I guess it wouldn't hurt, but I don't think it's necessary.

Reply:
ok lets leave it as it is for now, since it has been there anyway :-D